bukuserver/templates/bukuserver/statistic.html

```diff
@@ -4,25 +4,26 @@
 {% block head %}
   {{ super() }}
   {{ buku.close_if_popup() }}
-  <script>// realtime redrawing "Data created" datetime
+  <script>{// realtime redrawing "Data created" datetime
     const UNITS = {day: 60*60*24, hour: 60*60, minute: 60, second: 1};
+    const AGO = {{ {'day': _('{} days ago'), 'hour': _('{} hours ago'), 'minute': _('{} minutes ago'), 'second': _('{} seconds ago')}|tojson }};
```

Comment:
This isn't quite how translations with numbers are meant to be used, but this code runs in the browser.